packages/babel-preset-expo/src/common.ts

```diff
@@ -1,3 +1,7 @@
+import { type NodePath } from '@babel/core';
+// @ts-expect-error: missing types
+import { addNamed as addNamedImport } from '@babel/helper-module-imports';
```

Comment:
Could this be caused because we are implicitly depending on an outdated `@babel/helper-module-imports`?

Reply:
`@babel/helper-module-imports` is in the package.json, it doesn't have types.